<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralWidget</class>
 <widget class="QWidget" name="GeneralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>399</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>399</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="kcfg_ShowStop">
       <property name="text">
        <string>Show stop button</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="kcfg_ControlsOnBar">
       <property name="text">
        <string>Playback controls on the bar</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="kcfg_ButtonsAppearance">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Buttons appareance</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="flat">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>F&amp;lat</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="toolbutton">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;Tool button</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="kcfg_BackgroundHint">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Background hints</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="kcfg_NoBackground">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Custom colors</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="customcolors">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="KColorButton" name="kcfg_BackgroundColor">
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <property name="alphaChannelEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KColorButton" name="kcfg_FrontColor">
             <property name="flat">
              <bool>true</bool>
             </property>
             <property name="alphaChannelEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="frontcolor">
             <property name="text">
              <string>Front color</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="backgroundcolor">
             <property name="text">
              <string>Background color</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="kcfg_Normal">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;Normal</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="kcfg_Translucent">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Transl&amp;ucent</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_NoBackground</sender>
   <signal>toggled(bool)</signal>
   <receiver>customcolors</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
